sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -136,11 +145,45 @@ void scheduleMessage() {
             .verifyComplete();
     }
 
+    /**
+     * Verifies that we can schedule and receive multiple messages.
+     */
+    @Test
+    void testSendMultipleSceduledMessageAndReceive() {
```

Comment:
Spelling mistake: schedule

Reply:
Why prefixing with "test"? It's implied that `"@Test"` is a test.

Reply:
I copied these names from track1 as it is, and because did not want to miss any one of them, I kept names how it is in track 1, Let me change them now.